cves/kernel/CVE-2017-5548.yml

```diff
@@ -337,9 +353,12 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: False
+  stacktrace_with_fix: False
+  note: |
+    I tried checking all commits and CVE descriptions as well as dev notes. I did not
```

Comment:
Consider changing this to the third person.  Consider expanding dev developer for formal documentation.